### DISCUSSION TITLE
Allow refreshing an issue to work outside of a git repo

### DISCUSSION THREAD

**Discussion by ParetoOptimalDev** [magit/forge#695] [General]
I've added a single issue from a git repo. I navigate to it, post a comment to it from my web browser, and hit 'g'.

I get `magit--toplevel-safe: Not inside Git repository`.

I assume refresh of issues only works if you've cloned the repo? 

Would it be easy to allow refreshing of issues outside of git repos? I think it would be a useful feature.

**Comment by ParetoOptimalDev** [comment]
It looks like calling `forge-pull-topic` inside of the forge issue does what I want. Maybe rebinding `g` to `forge-pull-topic` in `forge-pull-topic` would be good.

**Reply by tarsius** [reply]
> Maybe rebinding `g` to `forge-pull-topic` in `forge-pull-topic` would be good.

Doing that would be equivalent to `g` in `magit-status-mode` doing `git fetch`, which wouldn't be good because it would always be slow and, when done so frequently, would only actually make a difference in a tiny number of cases. While doing "a full refresh, involving talking to a server" might make a bit more sense in a topic buffer, doing so would lead to inconsistency (magit/forge users would no longer be able to rely on `g` always being just a local operation).

**Comment by tarsius** [comment]
Fixed in 9b1bb42ca30b926d2ca537d85e3748fce0a034f3. Sorry this took so long.

**Reply by ParetoOptimalDev** [reply]
No problem. Thank you!
